build.gradle.kts

```diff
@@ -105,6 +106,12 @@ val copyCurrentDistro by task<Copy> {
     into(customInstallationDir)
     exclude("**/*kotlin*")
 
+    // TODO:pm remove SPIKE
+    from(file("gradle/gradle-api-parameter-names-SPIKE.jar")) {
+        into("lib")
+        rename { it.replace("SPIKE", GradleVersion.current().baseVersion.version) }
+    }
```

Comment:
❗️This would be generated by https://github.com/gradle/gradle/commit/15a9e245e55